src/components/layout/Header/HeaderV2.jsx

```diff
@@ -0,0 +1,122 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import { useDispatch } from 'react-redux'
+import { NavLink } from 'react-router-dom'
+import { reinitializeData } from 'redux-saga-data'
+
+import { signout } from 'repository/pcapi/pcapi'
+
+import Logo from '../Logo'
+
+import { ReactComponent as BookingsSvg } from './assets/bookings.svg'
+import { ReactComponent as CounterSvg } from './assets/counter.svg'
+import { ReactComponent as HomeSvg } from './assets/home.svg'
+import { ReactComponent as OffersSvg } from './assets/offers.svg'
+import { ReactComponent as RefundsSvg } from './assets/refunds.svg'
+import { ReactComponent as SignoutSvg } from './assets/signout.svg'
+import { ReactComponent as StyleguideSvg } from './assets/styleguide.svg'
+
+const HeaderV2 = ({ isStyleguideActive }) => {
+  const dispatch = useDispatch()
+
+  function onSignoutClick() {
+    signout().then(() => {
+      dispatch(reinitializeData())
+    })
+  }
+
+  return (
+    <header className="menu-v2">
+      <nav>
+        <div className="nav-brand">
+          <Logo className="nav-item" />
+        </div>
+
+        <div className="nav-menu">
+          <NavLink
+            className="nav-item"
+            role="menuitem"
+            to="/accueil"
+          >
+            <HomeSvg />
```

Comment:
```suggestion
            <HomeSvg aria-hidden />
```
Idem pour les autres